bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownButton.java

```diff
@@ -44,9 +44,11 @@
     private final ButtonList buttonListView;
     private final WebMarkupContainer baseButton;
     private final Icon icon;
-    private final IModel<AlignmentBehavior.Alignment> alignment = Model.of(AlignmentBehavior.Alignment.NONE);
-
-    private boolean dropUp = false;
+   
+    private final IModel<DropDownAlignmentBehavior.Alignment> alignment = Model
+			.of(DropDownAlignmentBehavior.Alignment.NONE);
+	private final IModel<DropDownVariationBehavior.Variation> variation = Model
```

Comment:
Leading whitespaces are missing. Please format it as the rest of the file.

Reply:
Done.